docs/rest_api.rst

```diff
@@ -683,6 +683,8 @@ Payments
 
    :reqjson int amount: Amount to be sent to the target
    :reqjson int identifier: Identifier of the payment (optional)
+   :reqjson int lock_timeout: lock timeout, in blocks, to be used with the payment. Default is 2 * channel's reveal_timeout (optional)
```

Comment:
Maybe add a note below talking about the limitations of the value provided. You do mention what the default is, but it's not clear why the value has to be larger than the reveal timeout. Please clarify it in a note below this line.

Reply:
Added that to the lock_timeout description. I prefer not to make changes to the .rst as PyCharm fails to display the result properly. If important, please help with this note.

Thanks @rakanalh